draft-ietf-webtrans-http2.md

```diff
@@ -784,6 +784,29 @@ WebTransport Data
                                     WebTransport Data
 ~~~
 
+# WebTransport Header Fields
+
+WebTransport over HTTP/2 uses HTTP header fields to communicate the initial
+values of the flow control windows, similar to how QUIC uses transport
+parameters.  All of the fields defined in this section are Item Structured
+Fields (Section 3.3 of {{!RFC8941}}); all of the field values MUST be Integer.
+If any of the fields cannot be parsed correctly or does not have a correct
+type, the peer MUST reset the CONNECT stream.
```

Comment:
Do we reset or reject with some error code/response?

Reply:
Usual way to handle this is to say ", the peer MUST treat the message as malformed' then the version-specific rules kick in